src/whisper_ctranslate2/whisper_ctranslate2.py

```diff
@@ -553,10 +561,11 @@ def main():
         local_files_only,
     )
 
+
     for audio_path in audio:
         if verbose and len(audio) > 1:
             print(f"\nFile: '{audio_path}'")
-
+            
         result = transcribe.inference(
```

Comment:
Please install black and run "pip python -m black *.py" before commit this to make sure that formatting is good and unnecessary additional spaces are removed if not needed.